app/controllers/home_controller.rb

```diff
@@ -0,0 +1,5 @@
+class HomeController < ApplicationController
+  def index
+    @users = User.all
+  end
+end
```

Comment:
- [ ] You're only required to create users and posts controller with index and show actions in both of them. Kindly take this index action to the users controller and remove this homeController

Reply:
Without a HomeController, how will I have an root page?

Reply:
On the wireframe, there isn't a /users route, the users are listed on the root route

Reply:
On the wireframe, there isn't a /users route, the users are listed on the root route